modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/Stresses.md

```diff
@@ -135,12 +116,13 @@ classes of UserObjects are as follows:
 - `HEVPStrengthUOBase`
 
 
-## Creep and Plasticity with StressUpdate Materials
+## StressUpdate Materials
+
+A set of plasticity and creep material models have been developed with the `StressUpdateBase` class,
+which allows for iterations within the material itself.These iterative
+materials are designated by `StressUpdate` at the end of the class name.
 
-In addition to the User Object based plasticity models, another set of plasticity and creep material
-models have been developed with the `StressUpdateBase` class, which allows for iterations within the
-material itself.  These iterative materials are designated by `StressUpdate` at the end of the class
-name. These classes use a Radial Return von Mises, or J2, approach to determine the inelastic, creep,
+These classes use a Radial Return von Mises, or J2, approach to determine the inelastic, creep,
```

Comment:
These classes can use a variety of approaches to determine the inelastic strain increment at each step. 

(they don't just have to be J2)